### PR TITLE
added new question on chmod in linux

### PR DESCRIPTION
**Pull request by PranjalSK03**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
 const linuxQuiz = [
   {
+    
+    Question: "Say a file named 'list' has has intial file permissions as '-rwxrwxrwx', A person wants to change its permissions using 'chmod' such that GROUP and OTHER can only read it, neither write or execute it?",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:4]: This question is a little to long. Let's shorten it. 
```suggestion
    Question: "In Linux, which command is used to remove the writing and executing permissions for 'group' and 'other'?",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+    Answer: "chmod go-wx list",
+    Distractor1: "chmod go+wx list",
+    Distractor2: "chmod a-wx list",
+    Distractor3: "chmod go+r list",
+    Explanation:
+      "'chmod go-wx list' will make 'g' group and 'o' other, '-' remove 'w' the writing and 'x' the executing permission only",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:10]:
```suggestion
      "'The chmod go-wx list is used to remove the writing and executing permissions for 'group' and 'other' for a file named list.",
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+    Link: "https://linuxize.com/post/chmod-command-in-linux/",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:11]: Please use freeCodeCamp resource here

File: src/stylesheets/HomepageRow.css [hunk 4]
```diff
@@ -64,6 +64,10 @@ p {
   padding-left: 10px;
 }
 
+/* ============================================= */
+/*              Responsive Browsing              */
+/* ============================================= */
+
```
Comment (jdwilkin4) [inline src/stylesheets/HomepageRow.css:70]: Please remove this comment since this commenting style has not been established in other parts of the app 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by PranjalSK03 — COMMENTED** [review]
updated the question in accordance to requirements

**Comment by PranjalSK03** [comment]
please see if any further changes required.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**